int64 lenpoints
geometry_msgs/Vector3[] points